auth/server/src/main/java/com/seagates3/response/formatter/xml/XMLResponseFormatter.java

```diff
@@ -177,6 +177,16 @@
       return success(operation);
     }
 
+    @Override public ServerResponse formatAttachUserPolicyResponse(
+        String operation) {
+      return success(operation);
+    }
```

Comment:
This is not required as AttachUserPolicy and DetachUserPolicy doesnt return any specific response back.
You just need to return 200 on success.